examples/cli/crud_partition.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+echo "CRUD partition script."
```

Comment:
I suggest to change this echo to something like "Example script demonstrating zhmc CLI partition commands."
Just in case users are not familiar with the acronym "CRUD".